azure/azure-pr-pipeline.yml

```diff
@@ -1,60 +1,42 @@
 name: "$(SourceBranchName)+$(BuildID)"
 
-# Prevents building every commit or on PR
 trigger: none
 pr: none
```

Comment:
presume this will need a PR trigger at some point ?

Reply:
Not needed